packages/page-anchor/src/Batch.tsx

```diff
@@ -54,22 +54,23 @@ function Batch(): React.ReactElement {
       return;
     }
 
-    const { compute_root, create_proof } = await import('mrklt');
+    const { construct } = await import('mrklt');
```

Comment:
This is the new optimized merkle tree constructor. It generates root and computes proofs at the same time. It reduces the number of hashes required.